x-pack/docs/en/security/operator-privileges/configure-operator-privileges.asciidoc

```diff
@@ -0,0 +1,53 @@
+[role="xpack"]
+[testenv="enterprise"]
+[[configure-operator-privileges]]
+=== Configure operator privileges
+
+==== Enable operator privileges
+Operator privileges need to be enabled explicitly on each node in the cluster.
```

Comment:
What happens if a node doesn't enable operator privileges while others are doing?

Reply:
You will get inconsistent result depending on which node the request hit first and where the request gets executed. Assuming the user has sufficient RBAC privileges for an action:
1. If the request hits a node with operator privileges turn on, it should generally work correctly. We could have 2 scenarios:
    * if the user is not an operator, the request will be rejected on this node and never get passed to another node.
    * if the user is an operator, the request will be authorized by the local node. When the request gets passed to another node where operator privileges are disabled, it will still work because it just become a normal request.
2. If the request hits a node with operator privileges disabled, things could break. The request is always authorized locally. Now:
    * if the request can be executed locally, it will be and things break.
    * if the request needs to be passed to another node where operator privileges are enabled, the remote node most likely will reject the request so that things still kinda work. A theoretical possibility is that the child action sent to the remote node is different from the parent action and it may or may not be operator-only. We don't have such issue right now, but if it becomes the case in the future, things will get more inconsistent.

Reply:
Thanks for the details. Would it make sense to explicitly warn that you can get an inconsistent behavior otherwise? Just to reinforce that.

Reply:
Added a warning about it. But didn't go into the details because it may be too distracting and the important message is just "please do not it".

Reply:
Thanks, LGTM! 👍